scripts/run_example.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+import argparse
+import math
+import os
+import psutil
+import subprocess
+import sys
+
+try:
+  import termcolor
+  canColor = True
+except ImportError:
+  canColor = False
+
+def logFile(jsonFile):
+  base = os.path.splitext(os.path.basename(jsonFile))[0]
+  return '/tmp/supersimtest_{0}'.format(base)
```

Comment:
optional: use tempfile stdlib module (with the context manager, you don't need to use os.remove() later)

Reply:
done